src/components/ContentSidebar/SkillsSidebar.js

```diff
@@ -38,12 +38,40 @@ class SkillsSidebar extends React.PureComponent<Props, State> {
         api.getMetadataAPI(false).getSkills(
             file,
             (cards: Array<SkillCard>) => {
+                this.updatePreviewTranscript(cards);
                 this.setState({ cards });
             },
             noop
         );
     }
 
+    /**
+     * Updates Preview with transcript data
+     *
+     * @private
+     * @param {Array<SkillCard>} cards - Skills cards
+     * @return {void}
+     */
+    updatePreviewTranscript = (cards: Array<SkillCard>) => {
+        const { getPreviewer } = this.props;
+        const preview = getPreviewer ? getPreviewer() : null;
```

Comment:
probably just call it getPreview() ? Since now its returning the preview object.
Before it was (Pre[view)er] lol